osgbuild/svn.py

```diff
@@ -177,13 +177,13 @@ def restricted_branch_matches_target(branch, target):
             target_osgver = target_match.groupdict().get("osgver", None)
             branch_osgver = branch_match.groupdict().get("osgver", None)
             if branch_name == "main":
-                branch_osgver = MAIN_OSGVER
+                branch_osgver = "3.5"
             if target_name == "main":
-                target_osgver = MAIN_OSGVER
+                target_osgver = "3.5"
             if branch_name == "upcoming":
-                branch_osgver = branch_osgver or UPCOMING_OSGVER
+                branch_osgver = branch_osgver or "3.5"
             if target_name == "upcoming":
-                target_osgver = target_osgver or UPCOMING_OSGVER
+                target_osgver = target_osgver or "3.5"
```

Comment:
Eh, why not just define the `MAIN_OSGVER` and `UPCOMING_OSGVER` constants _once_ in this function's scope?

Reply:
Eh. Maybe I'll call the constant `THREE_FIVE` :-P